.stylecop/GlobalSuppressions.cs

```diff
@@ -6,11 +6,14 @@
 // Project-level suppressions either have no target or are given
 // a specific target and scoped to a namespace, type, member, etc.
 
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements must be documented", Justification = "Test code does not require detailed documentation")]
```

Comment:
The way it showed up in VS seemed like it was only for the project, but that makes sense since it's for all projects. I removed the additional warning suppressions.

Looks like the `FunctionAppCorrelation` project has a package dependency on `StyleCop.Analyzers`, but some other samples that I checked don't have that dependency that's why this sample builds with a lot of warnings.

FunctionAppCorrelation.csproj
```
<PackageReference Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="All"/>

<ItemGroup>
    <AdditionalFiles Include="..\..\..\.stylecop\stylecop.json" />
    <Compile Include="..\..\..\.stylecop\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
  </ItemGroup>
```

Reply:
@ConnorMcMahon I reverted the GlobalSuppressions.cs file and removed additional warning suppressions. In general, do we want all of the samples to follow stylecop suppressions? Right now, some samples don't follow any stylecop rules. If we do, then I can create a new GitHub issue to track that and maybe the samples can follow their own stylecop suppression rules.

Reply:
@bachuv, I don't have a strong preference for our samples to file stylecop, but it's probably for the best for consistency across the repo. I am fine with opening a separate issue in the interest of getting these current improvements out the door.

Reply:
@ConnorMcMahon, I created issue #1799 to track this work. Is there anything else I should add/remove to this PR or is it good to merge?